electron-1/src/bx.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const { ipcRenderer } = require('electron');
+
+const bx = {
+  /**
+   * Generic promise gen to dialog with main/parent process
+   * @param {string} info | info aboout app to get [name,version..]
+   */
+  getInfos(info) {
+    return new Promise((resolve, reject) => {
+      ipcRenderer.send('getAppInfo', info);
+      ipcRenderer.on(info, (response, result) => {
```

Comment:
It looks to me you are attaching a new callback to `version` (respectively `name`) in any call to `getVersion` (resp. `getName`) without detaching it, no?

This could lead to [EventEmitter memory leak](https://nodejs.org/docs/latest/api/events.html#events_emitter_setmaxlisteners_n), and I guess, you are resolving your promise multiple times.

Reply:
Hum ys that clearly a memory leak!

I fixed the PR to prevent this